minecraft_servers/opinsel/manifest.json

```diff
@@ -5,15 +5,28 @@
     "server_wildcards": [
       "%.opinsel.net"
     ],
+    "gamemodes": {
+        "smp": {
+          "name": "TNTRun", 
+          "color": "#Ff0000",
+          "command": "/server tntrun",
+          "versions": "1.21<*"
+        }
+    },
+    "discord": {
+        "server_id": 1271149953221263452,
+        "rename_to_minecraft_name": false
+      },
     "supported_languages": [
         "de"
     ],
     "social": {
         "web": "https://opinsel.net",
         "web_shop": "https://opinsel.net/shop",
-        "discord": "https://discord.gg/Mn9drhfaDQ"
+        "discord": "https://discord.gg/SaBbwpT9hK"
     },
     "location": {
+        "city": "Eygelshoven",

```

Comment:
(Eygelshoven is in the Netherlands btw)

Reply:
![image](https://github.com/user-attachments/assets/2eeab421-e46b-4fe3-8eea-40c721b1a70d)
Not for my Hosting and not for https://github.com/LabyMod/server-media/blob/master/minecraft_servers/abbelsmp/manifest.json

Reply:
ZapHosting formulates it this way because it is officially only a "branch office". But that doesn't change the fact that Eygelshoven is in the Netherlands. See https://en.wikipedia.org/wiki/Eygelshoven , ZapHosting also describes it a little in their own wiki, see: https://zap-hosting.com/guides/de/docs/firststeps-locations/

But this is just a side note, because it caught my eye earlier.